README.md

```diff
@@ -8,3 +8,267 @@ To enable one of the scripts you have to add a line like `require "official/hell
+
+Adds a dropdown list with import filters to the import dialog. Several import filters can plug into the manager. This allows customized import schemes such as “prefer raw over jpeg” where a jpeg is only added if no corresponding raw file exists. Requires suitable filters, an example set is implemented in “import_filters.lua”.
+
+
+### import_filters.lua
+
+*Compatibility: unknown*
+
+Implements suitable import filters to be used with “import_filter_manager.lua”. Two filters are available, one that resembles the “ignore jpeg” functionality of darktable's import dialog and one that implements the “prefer raw over jpeg” strategy explained in the import_filter_manager.lua description.
+
+
+### save_selection.lua
+
+*Compatibility: 1.x, 2.x*
+
+Provides shortcuts to save selections to and restore them from up to five temporary buffers.
+
+
+### selection_to_pdf.lua
+
+*Compatibility: 1.x, 2.x*  
+*Tags: storage*
+
+Register a new exporter that exports selected images into a single PDF file by using LaTeX. Requires a proper LaTeX installation.
+
+
+### yield.lua
+
+*Compatibility: unknown*
+
+Some compatibility code, but for what exactly?
+
+
+## Contributed
+
+The scripts of this section are maintained by individual people, which are listed in this overview and are as well mentioned in the individual scripts (there's the more accurate information). Please try to contact the maintainers for bug reports first.
+
+
+### autostyle.lua
+
+*Compatibility: 1.x, 2.x*  
+*Maintainer: Marc Cousin (cousinmarc@gmail.com)*
+
+Automatically apply a given style when an exif tag is present in the file, e.g. to apply a style to compensate for Auto-DR from some Fujifilm cameras.
+
+
+### calcDistance.lua
+
+*Compatibility: 2.x*  
+*Maintainer: Tobias Jakobs*  
+*Tags: geo*
+
+Calculate the distance between the places where two images were taken by using the GPS metadata.
+
+
+### copy_attach_detach_tags.lua
+
+*Compatibility: 2.x, 3.x*  
+*Maintainer: Christian Kanzian*  
+*Tags: tagging*
+
+Copy, paste, replace and remove tags from images by shortcuts and a distinct lighttable module.
+
+
+### cr2hdr.lua
+
+*Compatibility: 1.x, 2.x*  
+*Maintainer: Till Theato (theato@ttill.de)*  
+*Dependencies: cr2hdr program from Magic Lantern (http://www.magiclantern.fm/forum/index.php?topic=7139.0)*
+
+Process images shot with Magic Lantern's Dual ISO feature for dynamic range improvement with the cr2hdr program from lighttable and import the result (with shortcut and/or on import).
+
+
+### geoJSON_export.lua
+
+*Compatibility: 2.x*  
+*Maintainer: Tobias Jakobs*  
+*Dependencies: mkdir, convert (ImageMagick), xdg-open, xdg-user-dir*  
+*Tags: geo*
+
+Generate GeoJSON file from image metadata. **What exactly is exported? What is a typical use case?**
+
+
+### geo_uri.lua
+
+*Compatibility: 2.x*  
+*Maintainer: Tobias Jakobs*  
+*Dependencies: gnome-maps ≥ 3.20*  
+*Tags: geo*
+
+Open a geo uri in gnome-maps. **Where is this URI found? What is a typical use case?**
```

Comment:
This will open the location of the image with the standard program for geo uri. E. g. GnomeMaps supports this:
http://jonasdn.blogspot.co.uk/2015/12/whats-cooking-in-maps-in-gnome-320.html
https://en.wikipedia.org/wiki/Geo_URI_scheme